### ISSUE TITLE
Rating for developers etc

### ISSUE THREAD

**Issue by heinvv** [Forket/connect2sa.co.za_01#7]
Documents not ready yet
We are working on this.

**Comment by Forket** [comment]
OK. assign to U

**Comment by heinvv** [comment]
Please find description in document attached
[20151026 advanced ratings.docx](https://github.com/Forket/connect2sa.co.za_01/files/37880/20151026.advanced.ratings.docx)

**Comment by Forket** [comment]
This is real hell in code :) I even can't imagine how you plan go it live. Sorry for truth.

**Comment by Forket** [comment]
Ratng ready.
Please check. 
http://www.connect2sa.co.za/rigel/members/admin/

**Comment by heinvv** [comment]
Great.

Can you please after the 5 stars add "x reviews", where x is the number of
reviews.

**Comment by heinvv** [comment]
There seems to be a problem with reviewing websites.

Every website should have their unique review section (e.g. website A get
3,4 and 3 stars, website B 2 and 5 star rating etc).

Right now I get the impression that the review section is the same for all
websites.

Please see the bottom section. Both have the same reviews displayed.
http://www.connect2sa.co.za/rigel/listing/myyyyytestttttt/
http://www.connect2sa.co.za/rigel/listing/listings-four/

**Comment by heinvv** [comment]
For an example of how the rating function should work:

http://topgauge.co.za/listing/555-liberty-village-toronto-ontario-canada-astoundify-barbershop/

Login Details

Wordpress
Topgauge.co.za/index.php
User admin
Passw Future2015

Topgauge.co.za/wp-admin
User admin
Passw Future2015

Webhost
1. Package Details:
==================================================
Package: Business (Local)
Domain: topgauge.co.za
Current Email Contact: kobuscoetzee1971@gmail.com
Username : topgauge
Password : Future2015
Server: sh01-pta
Server IP address: 196.46.186.134
2. Website/Domain and Nameserver details:
==================================================
All the following website addresses will work for your domain once the
necessary DNS has propagated (approximately 24 hours):
Temporary Address 1: http://196.46.186.134/~topgauge/
Live Address 1: http://www.topgauge.co.za
Live Address 2: www.topgauge.co.za
Live Address 3: topgauge.co.za
Note: some scripts and installed software might not function/display
correctly via the temporary address.
The details for our nameserver/dns cluster are below, you will need this
information when registering or updating a domain with an external
registrar.
Primary DNS: ns3.za-dns.com
Primary DNS IP: 196.2.147.183
DNS Server Location: Johannesburg, South Africa
Secondary DNS: ns5.za-dns.com
Secondary DNS IP: 216.18.193.151
DNS Server Location: Los Angeles, USA
3. FTP (Publishing) Details
==================================================
FTP Host: www.topgauge.co.za or 196.46.186.134
FTP Port: 21
Username : topgauge
Password : Future2015
Please ensure that your home page is named index.html, index.htm or
index.php (all small letters). This will supercede the existing
elitehost.php 'coming soon'.
Please do not delete any of the existing folders in your FTP account or
change any permissions as it can break your email and control panel
functions.
If you are publishing using any FTP client then follow these instructions:
1. Login to your FTP account using the details above
2. Go into (i.e. double click) the "public_html" directory
3. Transfer/Upload the applicable website files into this directory
Video Tutorials: http://my.elitehost.co.za/tutorials.php?series=52
4. Control Panel (cPanel) Login Details:
==================================================
Go to URL: http://topgauge.co.za/cpanel
Username: topgauge
Password : Future2015
You can access cPanel at any time in order to perform the following tasks:
- create/edit email addresses
- setup databases
- view your website statistics
- access sitebuilder (under Software)
- access the auto-installer for software such as WordPress, Joomla,
OSCommerce etc
and other advanced fetures such as:
- reviewing error logs to locate broken links and other problems relating
to any internal server errors caused by incorrect permissions.
- generate ssl certificate signing request (csr)
- create backups Please note that your domain may take 24
- 48 hours to appear online due to propagation time.
You can access your cPanel immediately by clicking on the following link:
http://196.46.186.134:2082/login/?user=topgauge&pass=Future2015
You can also use cPanel over secure (ssl) connection:
https://sh01-pta.za-dns.com:2083
If you cannot access port 2082 because of a work proxy etc you can use
http://cpanel.topgauge.co.za - certain features might not function
correctly though, but you'll be able to setup mail accounts and view stats.
Video Tutorials: http://my.elitehost.co.za/tutorials.php?series=289
5. Email/Mailbox Details:
==================================================
You can setup new email addresses (mail boxes) or forwarders in your
Control Panel (cPanel). Please login using the login details found in the
section above.
Once you have logged in click "Mail Accounts" under the Mail section of
cPanel. You can view the virutal turial "How to create a POP email account
in cPanel" under the "cPanel x3 end-user" section of our video tutorials.
Our Unlimited hosting allows for an unlimited number of email addresses to
be created, each email address allows for unlimited storage. Once you have
setup new email addresses you can send and retreive email with a mail
client/program or mobile service of your choice.
Please use the details below:
Incoming Mail Server/host (POP3/IMAP): mail.topgauge.co.za
Port: 110 for POP3 and 143 for IMAP
Outgoing Mail Server/host (SMTP): mail.topgauge.co.za
Port: 26
Username: full email address ie. myname@topgauge.co.za (it must be your
full email address including the domain name.)
Password: the password you entered in cPanel when creating the email
account.
Note: we do run an outgoing mail server on standard port 25, but we highly
recommend using the mailserver on port 26 or 587 as this eliminates most
filtering done by ISPs.
We recommend using IMAP to receive emails as this will sync your client
between our servers, allowing you to access your emails via webmail and
storage a copy of each email on our servers. If you do not remember your
mailbox password then you can reset it in cPanel.
Please note that we do not allow any form of spam to be sent via our
servers. Abusers will be suspended.
For further help on configure an email address in outlook/thunderbird etc
please view our virtual tutorials here -
http://my.elitehost.co.za/tutorials.php?series=40
6. Webmail Details:
==================================================
Your emails can also be viewed and sent online directly via your webmail
facility that you are provided with.
To use the access the webmail facility please use the following details:
URL: http://topgauge.co.za/webmail/
Username: full email address ie. myname@topgauge.co.za (it must be your
full email address including the domain name.)
Password: the password you entered in cPanel when creating the email
account.
We recommend using the RoundCube interface, as this is the most user
friendly and modern.
If you do not remember your password then you can reset it in cPanel.
You can also use webmail over secure (ssl) connection:
https://sh01-pta.za-dns.com:2096
If you cannot access port 2095 because of a work proxy etc you can use
http://webmail.topgauge.co.za
7. Technical Support Details:
==================================================
If you require assistance with any aspects of your hosting then please feel
free to submit a ticket to the "Technical Support" department.
Please submit a ticket for any technical support. Go to:
https://my.elitehost.co.za/submitticket.php
You can view any updates to a ticket. Go to:
https://my.elitehost.co.za/supporttickets.php
Please help us improve efficiency by using an appropriate and descriptive
subject when submitting a ticket.
You can also email helpdesk@elitehost.co.za or use our live chat function
on our website. Live chat is generally available during business hours and
should only used for very basic questions.
View over 100 video tutorials on how to use cPanel, Softaculous, FTP, Mail
and more on your hosting account.http://my.elitehost.co.za/tutorials.php
If you ever get blocked by our server firewall then you can unblock
yourself here - https://my.elitehost.co.za/firewall.php
If you require any futher information, please feel free to reply back to
this email.
8. Customer Service and Feedback Details:
==================================================
At Elitehost we strive to offer outstanding customer support. If at anytime
you feel that our technical support has not been up to standard then please
email management@elitehost.co.za
We also value your input, as we always looking for new ideas. Please let us
know if you have any by emailingmanagement@elitehost.co.za
Hellopeter: http://www.hellopeter.com/elitehost/compliments-and-complaints
Twitter: http://twitter.com/elitehostcoza
Facebook: http://www.facebook.com/elitehost
9. Mail Filtering (SpamExperts)
==================================================
You can access your SpamExperts control panel by clicking the "SpamExperts"
icon in cPanel under the Mail section. You can search your incoming and
outgoing mail logs as well as view filtering reports.
Your domain should be configured to use the following mail exchange (MX)
records:
mx1.zamailgate.com (Priority 10)
mx2.zamailgate.com (Priority 20)
Please contact support for further information regarding SpamExperts.
==================================================
Our business hosting environment is designed for speed and security.

**Comment by heinvv** [comment]
I realise that this is slightly more work than I had expected. I thought reviewing worked properly for separate listings and wpthemes, but apparently that is not the case. 

1.
Can you first please install reviewing section for every website listing separately (see above). 
2. 
After that can you please install reviewing section for every WPTHEMES page separately.
3. 
Afterwards can you please install the ratings for theme developer and theme seller pages? (That should be the average of the themes that are assigned to them. 
4. 
I think afterwards it is best to other issue about assigning website listings to developers. 
5. 
Afterwards, ratings for website developers; use the average of projects assigned to them; but now slightly more complicated.   
Please on't use the overall ratings for website listings, but you the "aspect ratings" (see word document; but we can discuss this later on). I think it is best to first steps 1-4, before we discuss 5.

**Comment by heinvv** [comment]
Can you please use these images for the ratings?
<img width="8" alt="stars" src="https://cloud.githubusercontent.com/assets/11960838/11466984/ccba3ff4-974d-11e5-8397-47e3e705e5e2.png">
<img width="8" alt="stars_half" src="https://cloud.githubusercontent.com/assets/11960838/11466983/ccba3838-974d-11e5-895e-d0bb998d33a8.png">
<img width="8" alt="stars_off" src="https://cloud.githubusercontent.com/assets/11960838/11466985/ccbee2a2-974d-11e5-9167-7b9c18f5ebb0.png">

**Comment by Forket** [comment]
created separate issue #14  for fixing reviews

**Comment by Forket** [comment]
>Can you please use these images for the ratings?

But they are the same? where is the difference ?

**Comment by heinvv** [comment]
I think the new ones are slightly clearer/brighter

**Comment by Forket** [comment]
Current start are used in any places at your solution and they use plugin
for that jquery-ui-stars (https://github.com/garrett/jquery.ui.stars). I
can use start suggested by you but you should know that:
1. They will be only in my solution (rating for developers)
2. it's separate issue because It needs custom coding.
Please confirm with YES or NO.


Thanks and regards,

*Aleksandr Sen*

CEO | Expedition PM

Mobile: +48 532 292 764

Skype: aleksandr.sen

Email: alex@sen.pm

Social: LinkedIn <http://ua.linkedin.com/in/aleksandrsen/> | Google+
<https://www.google.com/+AleksandrSen>



*EXPEDITION.PM <http://www.expedition.pm/> | KNOWLEDGEMAP.PM
<http://www.knowledgemap.pm/>*

2015-12-01 14:49 GMT+01:00 heinvv <notifications@github.com>:

> I think the new ones are slightly clearer/brighter
>

**Comment by heinvv** [comment]
I fear that there is some kind of confusion here.

We only want ONE rating / review system. Not two.

I get the impression that your Star rating is based on an extra
plugin. That can't work.

Please make sure that ALL ratings on the website are based on the Ratings
add to the system by clicking on "Add Review" on separate website listing
pages and on WP Theme pages.

I hope this is clear.

**Comment by Forket** [comment]
OK. Once again.
Someone, who developed rating used SEPARATE plugin jquery-ui-stars for rendering stars (https://github.com/garrett/jquery.ui.stars). Most of all, this plugin is 6 years old and currently not supported by owner.
![snapshot n001_2015-12-01_15-01-24](https://cloud.githubusercontent.com/assets/6735288/11502657/6889db30-983c-11e5-8300-d0bf5b664fb1.png)

The prove of that fact that this plugin is used by some of developers who worked with you is here:
![snapshot n001_2015-12-01_15-05-04](https://cloud.githubusercontent.com/assets/6735288/11502745/ef87b67a-983c-11e5-9c1f-a5924f1bcf4d.png)
![snapshot n001_2015-12-01_15-03-44](https://cloud.githubusercontent.com/assets/6735288/11502746/efa128f8-983c-11e5-8fdd-1e1d02f89660.png)

This means that:
- Make changes in plugin jquery-ui-start (faster)
OR
- use another plugin and integrate it into your theme (longer)

**Comment by heinvv** [comment]
Ah, I see.

Thanks for clearing that up.

Can I try to put my question differently.

1.
I see stars on http://www.connect2sa.co.za/rigel/members/admin/.
2.
I see different stars on http://www.connect2sa.co.za/rigel/wptheme/avada/.

We like the stars on page 2. Not on page 1.
That was the reason why we sent you the stars.

Can you please use the same stars for both 1 and 2?

**Comment by heinvv** [comment]
I just received email with this information:

"

header

Dear admin,

we are happy to review

Kind regards,
The rigel team

footer"

**Comment by Forket** [comment]
I've made changes to jauer-start plugin. you can check stars in 10 min.
Before check please clear cache.

**Comment by heinvv** [comment]
Perfect!

**Comment by Forket** [comment]
About reviews in profile. The number should show sum (count) of reviews which for example admin wrote. Right?

**Comment by Forket** [comment]
about reviews - done
For now admin has 3 reviews and it's shown in his profile http://www.connect2sa.co.za/rigel/members/admin/

is that what you wanted?

**Comment by Forket** [comment]
Please check  and is it's OK i'll close this issue and will move forward.

**Comment by heinvv** [comment]
This is only partially what we wanted. 

But I think you can only do this properly, once you have fixed reviews. 

Right now the ratings on the developer page are irrelevant, because it doesn't indicate this developers' work. Instead it is an average of all website ratings. 

I think what we should do first: 
- install separate review functionality on every separate website page an on every separate theme page. 

Afterwards we can continue with ratings for developers, theme developers and sellers.

**Comment by heinvv** [comment]
I think you have missed this from the advanced ratings word document: 

Rating for theme sellers
Calculate average based on ratings of themes sold by this seller. 

Display rating:
Display rating as 5 star rating on the theme seller page (e.g. http://connect2sa.co.za/rigel/themesellers/seller-1/) and mention the number of reviews. 

Rating for theme developers
Calculate average based on ratings of themes developed by this seller. 

Display rating:
Display rating as 5 star rating on the theme seller page (e.g. http://connect2sa.co.za/rigel/themedeveloper/developer-1/)and mention the number of reviews.

**Comment by heinvv** [comment]
Calculation of ratings seems not to be working on Theme page. 

E.g. screenshot (http://www.connect2sa.co.za/rigel/listing/again/)

5 aspects are reviewed.
All with maximum score. 
However, the average for that theme is 4/5. 

Calculation should be like this: 
Select average of those aspects that are rated. 
E.g. 
4 aspects rated. Calculate the average of those 4 aspects. 


![image](https://cloud.githubusercontent.com/assets/11960838/11507110/2a7a85f0-985b-11e5-891e-5b6e8e8da875.png)

**Comment by heinvv** [comment]
I don't think you have used all of this information: 

Rating for developers
We’d like to calculate the rating for developers indirectly.
We want to base their rating on their website ratings.
E.g.
Tim Johnson is a graphic designer. 
He designs websites, but also sometime works as the “creative mind”. 

Tim’s websites and ratings: 
•	Abc.com	rating (designer): 4/5					average: 4.5/5
•	Def.com	rating (designer): 3/5					average: 3/5
•	Ghi.com	rating (designer): 4/5	rating (creative mind): 5/5	average: 4.5/5
Rating per website: 
•	Abc.com	rating: 4/5
•	Def.com	rating: 3/5
•	Ghi.com	rating: 4.5/5		use the average of all ratings per website
Average ratings:
(sum of all ratings) / (number of ratings)
= (4+3+4.5)/3
= 11.5/3
= 3.83
= 3.8 / 5
=7.6 / 10

**Comment by heinvv** [comment]
In short
We first need to fix reviews, before we can implement advanced ratings.

**Comment by Forket** [comment]
Ok. agreed.
I'd like again figure out terminology.
What is the difference between:
- Listing (ex. http://www.connect2sa.co.za/rigel/listing/test/)
- Website 
- Theme (ex. http://www.connect2sa.co.za/rigel/wptheme/avada/)

Could you be so kind and give me a link for **Website** example? or website it's the same as listing?

**Comment by Forket** [comment]
>I don't think you have used all of this information:

I exactly used info from document. But for now just completed rating for developers.

**Comment by heinvv** [comment]
Listing = website listings = website

**Comment by heinvv** [comment]
"I exactly used info from document. But for now just completed rating for developers." 
Great.

**Comment by Forket** [comment]
>I don't think you have used all of this information:

In order to know you should know that in order to determinate who is developer i'm looking here:
http://www.connect2sa.co.za/rigel/wp-admin/post.php?post=7931&action=edit

![snapshot n001_2015-12-01_17-48-47](https://cloud.githubusercontent.com/assets/6735288/11507323/c90fbf30-9853-11e5-9984-cce086c13874.png)

In this rating i'm looking for now for information about Designer.

**Comment by Forket** [comment]
am I right that in some sites admin (for example) can be programmer in some sites - designer , content writer etc.. So I have to take into consideration (for calculation) all the sites where admin is present. Right?

**Comment by heinvv** [comment]
That is correct.
And can have multiple roles in another project.

I hope that my examples from the word document were clear.

**Comment by heinvv** [comment]
I am logging off now.

Chat later tomorrow.

Thanks for all your work.
Sorry for the confusion because of other developers.

**Comment by Forket** [comment]
It's Ok. Don't worry. 
I like to solve problems :) this is my job title: "Problem Solver" :))

**Comment by heinvv** [comment]
Mr Master Problem Solver

-----Original Message-----

**Comment by Forket** [comment]
i'm already taking into consideration all possible roles ( designer, programmer, etc) at listings where admin is mentioned. So if listing has rating and admin is selected at some role - i'll calculate it.
For calculation I use this data:
![snapshot n001_2015-12-01_19-40-37](https://cloud.githubusercontent.com/assets/6735288/11510307/674f5408-9863-11e5-8079-31c15db2e8d9.png)

**Comment by Forket** [comment]
So reviews fixed. 
and also you can see change in reviews count in profile.
http://www.connect2sa.co.za/rigel/members/admin/
Is it what you wanted?

**Comment by heinvv** [comment]
I just check this page: http://www.connect2sa.co.za/rigel/listing/again/.

This seems to be working perfectly, right now.

Great work.

Can you please do the same for themes as well?(e.g.
http://www.connect2sa.co.za/rigel/wptheme/avada/).

**Comment by Forket** [comment]
>Can you please do the same for themes as well

themes don't have rating functionality. In order to have it it must be developed for this. It's separate issue.

**Comment by heinvv** [comment]
?

I meant on wpthemes pages.

I think you have fixed this already.

E.g. I see review button on this page:
http://www.connect2sa.co.za/rigel/wptheme/avada/

**Comment by Forket** [comment]
OK. Got it.

**Comment by Forket** [comment]
Can U confirm that it works?

**Comment by heinvv** [comment]
http://www.connect2sa.co.za/rigel/themesellers/seller-1/

http://www.connect2sa.co.za/rigel/themedeveloper/developer-1/

**Comment by heinvv** [comment]
http://www.connect2sa.co.za/rigel/wptheme/avada/: I have just edit a review, and it is displayed. 

Can you at the top of the page display a 5 star rating and a number of reviews as well?

**Comment by heinvv** [comment]
We'd like to create a rating, where it displays 5 star rating and 145 reviews. 

http://www.connect2sa.co.za/rigel/themesellers/seller-1/: can you please display the average rating of all the themes linked to this seller?

http://www.connect2sa.co.za/rigel/themedeveloper/developer-1/: : can you please display the average rating of all the themes linked to this seller?

**Comment by heinvv** [comment]
On http://www.connect2sa.co.za/rigel/members/admin/: can you please delete "8/10"?

**Comment by heinvv** [comment]
I am not sure if the rating on http://www.connect2sa.co.za/rigel/members/admin/ is correct. 
I just check it:
It had 6 ratings and showed 8/10. 
In total that is around 6*8=48 (out of 60). 

I added one rating (all aspects 1/5) on http://www.connect2sa.co.za/rigel/listing/myyyyytestttttt/. 
The new total for admin should be: 
48+2 = 50 (out of 70). 
50 out of 70 is around 70%. 

But on http://www.connect2sa.co.za/rigel/members/admin/ is still displays the score of 8/10 (and 4 stars). 

But this would still be possible because of rounding off. 

I thought, let me test again. 
I added a 1/5 for http://www.connect2sa.co.za/rigel/listing/watch/ as well. 

But the score is still 8/10 after 8 reviews. 

Can you please check this for me?

If there is a problem, can you please tell me how you worked out the rating?

**Comment by Forket** [comment]
>http://www.connect2sa.co.za/rigel/wptheme/avada/: I have just edit a review, and it is displayed.
Can you at the top of the page display a 5 star rating and a number of reviews as well?

It's not just about adding stars. your current solution doesn't support it. and is not scalable. So to do it I have to replicate ration functionality from other objects (for example listings). It's separate issue. I've opened new one #16

**Comment by Forket** [comment]
> On http://www.connect2sa.co.za/rigel/members/admin/: can you please delete "8/10"?

Done

**Comment by Forket** [comment]
>We'd like to create a rating, where it displays 5 star rating and 145 reviews.
http://www.connect2sa.co.za/rigel/themesellers/seller-1/: can you please display the average rating of all the themes linked to this seller?
http://www.connect2sa.co.za/rigel/themedeveloper/developer-1/: : can you please display the average rating of all the themes linked to this seller?

developers and themesellers (for example) have different implementation because of this it's out of scope of this issue. Because developers it's wordpress users and themesellers it's content type. This is because person who developed ratings have implemented wrong approach. 

it's in scope of issue #16

**Comment by Forket** [comment]
>I am not sure if the rating on http://www.connect2sa.co.za/rigel/members/admin/ is correct. 
I just check it:
It had 6 ratings and showed 8/10. 
In total that is around 6*8=48 (out of 60).

You added rating in wrong place. In fact you added review BUT in calculation I'm taking listing RATING.
Take a look at screenshot.
![snapshot n76_2015-12-03_13-02-53](https://cloud.githubusercontent.com/assets/6735288/11560118/6143ef7c-99be-11e5-8248-2ff37c98451f.png)

**Comment by heinvv** [comment]
Ah, I am sorry for the confusion. 

Sorry, I didn't think about this. 

Can you please use bottom one only. The one that I am using. 

And can you please delete the top one?

**Comment by heinvv** [comment]
Can you work with the bottom one version?

I only want Reviews + Ratings implemented (not those separate ratings).

**Comment by Forket** [comment]
OK. Based on your comments for issue #16. Let's close this issue and move to your updated version in Issue #16.

**Comment by heinvv** [comment]
I'd like to clarify one thing here.

This week you indicated that you used aspect rating used for listings to
find the rating for members (developers).

You said you had used all my instructions from the word document.

I am confused why you used the overall rating on top of the website then.

e.g. " ·       Ghi.com              rating (designer): 4/5     rating
(creative mind): 5/5              average: 4.5/5"

We specifically use rating aspects here.

**Comment by Forket** [comment]
Maybe you didn't know or previous developers worked separately, BUT in your system there are TWO rating solutions which work/live separetely from each other.
There are rating system which goes from **directory** plugin AND another rating system which does from theme.

**Comment by Forket** [comment]
And in the top - it's rating system from DIRECTORY plugin and ratings in REVIEWS it's another system.

**Comment by Forket** [comment]
So for now I understand that you want to use rating system at REVIEWS.
And this is a big bug to solve. because ratings in REVIEW works *only* for listings and themes and because of this main issue is to link this rating system to themesellers etc.

**Comment by heinvv** [comment]
I just wrote a message about Rating and Review system, but I can't find the
message right now. Maybe I didn't send it.

I said: We only use the Review system at the bottom of the page.
We don't use rating system from Listings.

Confusing part for me though:
You said you had used all my instructions from the word document.
But the focus is there for 100% on aspect rating from review section.
You even showed my screenshot from database, where you get the aspect
ratings for every different user.

I don't understand why this then comes as a surprise to you. I am really
confused.

About rating for Theme Developers and Theme Sellers: that was the 2nd page
of the word document "Advanced Ratings". I have asked you to develop this
from the beginning.

**Comment by Forket** [comment]
>You even showed my screenshot from database, where you get the aspect
ratings for every different user.

I've sent you screenshot as below :) and it's now rating from reviews. Because `user_designer` and such fields came from rating system in DIRECTORY plugin.
![c90fbf30-9853-11e5-9984-cce086c13874](https://cloud.githubusercontent.com/assets/6735288/11561655/18df6e68-99c9-11e5-8eba-c518e0faed8e.png)

**Comment by Forket** [comment]
In your description for rating you also used these user roles (designer, create mind etc) and these roles don't belong to REVIEWS rating system.

**Comment by heinvv** [comment]
This comes as a huge surprise to me.

I have never intended to use the Directory rating system.

The ratings aspects (designer; creative mind etc) come from me, and were
not part of the original directory rating system.

Are these aspect ratings not saved in the review section?

Still: the only place where you can use these aspects in the review panel.
So even if they are saved in directory rating it still should effect the
member rating.

I guess you have to do quite some recoding then, so that you can use the
same system both for Listings and for WP Themes.

Is the system as I envision it, clear to you now?

**Comment by Forket** [comment]
Deal. Now is clear. Anyway will ask you to test during the process of developing to be sure that it is what you want. 


Sent from Mobile

Thanks and regards,
Aleksandr Sen
CEO | Expedition PM
Mobile: +48 532 292 764
Skype: aleksandr.sen
Email: alex@sen.pm
Social: LinkedIn | Google+
 
EXPEDITION.PM | KNOWLEDGEMAP.PM

**Comment by heinvv** [comment]
Great

**Comment by heinvv** [comment]
I'd like to add one aspect:
We'd like to show an average of rating per aspect as well.
Please see REVIEW section on
http://www.connect2sa.co.za/rigel/test/profile/theme_profile.php
http://www.connect2sa.co.za/rigel/test/profile/pro_page.php.

**Comment by Forket** [comment]
Where is this page in WP-admin? (please answer in newly created issue #16  for ratings in order now to lost information)
